### PR TITLE
virttest/virt_vm.py: Update get_memory_size() for windows guest

### PR DESCRIPTION
**Pull request by spcui**
Update the method to get windows guest TotalMemory.

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1295311

### CODE REVIEW

**Comment by xutian** [comment]
Nack,  MaxCapacity mean OS support max memory size.  We are not always assign VM max support memory.

**Comment by CongSmile** [comment]
Agree w/ xu.

Maybe you can boot the guest with the limited memory for testing for this case.

Thanks.

File: virttest/virt_vm.py [hunk 1]
```diff
@@ -1287,13 +1287,16 @@ def get_memory_size(self, cmd=None, timeout=60, re_str=None):
             mem_size = 0
             for m in mem:
                 mem_size += int(m)
-            if "GB" in mem_str:
-                mem_size *= 1024
-            elif "MB" in mem_str:
-                pass
+            if self.params.get("os_type") == "windows":
```
Comment (xutian) [inline virttest/virt_vm.py]: we  have framework function utils_misc.normalize_data_size to convert it

**Comment by CongSmile** [comment]
@spcui 

Could you give an explanation about the way / idea to fix this bug? I'm a little confused.

Thanks.

File: virttest/virt_vm.py [hunk 2]
```diff
@@ -1287,13 +1287,19 @@ def get_memory_size(self, cmd=None, timeout=60, re_str=None):
             mem_size = 0
             for m in mem:
                 mem_size += int(m)
-            if "GB" in mem_str:
```
Comment (xutian) [inline virttest/virt_vm.py]: it looks 'normalize_data_size' can handle size string,  please test.

Comment (spcui) [inline virttest/virt_vm.py]: @xutian mem_str is return by "dmidecode -t 17 | awk -F: '/Size/ {print $2}'", not for "cat /proc/meminfo", so mem_str like this:
"
 4096 MB
 4096 MB
"
so, cannot use mem_str,  need to catch the unit.

**Comment by spcui** [comment]
PASS smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.physical_resources_check
PASS smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.RHEL.6.7.x86_64.io-github-autotest-qemu.physical_resources_check

**Comment by spcui** [comment]
@xutian updated with your comment, pls review it again.

**Pull request merged**